### PR TITLE
Fix buttons problems in 'Change Email' section.

### PR DESCRIPTION
**Pull request by dojutsu-user**
Problem it solves: 
1. Even if the email is verified and primary, buttons still show to resend verification link and to make the email primary.
2. If the email is not verified, button should not be there to make it as primary.
3. if the email is not verified and not primary, only show `Resend verification link` button and after it has been verified, hide `Resend Verification link` button and show `Make primary` button.
4. Also, there was no use-case for block `extra_body`. It should be `footerjs`.

Demo:

ezgif-2-8bd36bbb91cc

Currently, I have implemented this on client side, this should also be from server side. I was able to send 3-4 email verification email for a verified email.

### CODE REVIEW

**Comment by dojutsu-user** [comment]
@humitos 
Can I request a review here?

**Comment by dojutsu-user** [comment]
This is still valid bot.

**Comment by dojutsu-user** [comment]
Still valid bot.
Partially linked to #5665 
Relevant Comment: [#5665 (comment)](https://github.com/rtfd/readthedocs.org/issues/5665#issuecomment-490014465)

File: readthedocs/templates/account/email.html [hunk 1]
```diff
@@ -9,68 +9,104 @@
...
+    <p>{% trans 'The following email addresses are associated with your account:' %}</p>
+
+    <form action="{% url 'account_email' %}" class="email_list" method="post">
+      {% csrf_token %}
+      <fieldset class="blockLabels">
+
+        {% for emailaddress in user.emailaddress_set.all %}
+          <div class="ctrlHolder">
+            <label for="email_radio_{{forloop.counter}}" class="{% if emailaddress.primary %}primary_email{%endif%}">
+              <input id="email_radio_{{forloop.counter}}" type="radio" name="email" {% if emailaddress.primary %}checked="checked"{%endif %} value="{{emailaddress.email}}"/>
```
Comment (humitos) [inline readthedocs/templates/account/email.html]: It's a good practice to use spaces around `{{` and `{%`. For example, `{{ forloop.counter }}`.

File: readthedocs/templates/account/email.html [hunk 2]
```diff
@@ -9,68 +9,104 @@
...
-</div>
-  {% endfor %}
-
-<div class="buttonHolder">
-      <button class="secondaryAction" type="submit" name="action_primary" >{% trans 'Make Primary' %}</button>
-      <button class="secondaryAction" type="submit" name="action_send" >{% trans 'Re-send Verification' %}</button>
-      <button class="primaryAction" type="submit" name="action_remove" >{% trans 'Remove' %}</button>
-</div>
+  {% if user.emailaddress_set.all %}
+    <p>{% trans 'The following email addresses are associated with your account:' %}</p>
```
Comment (humitos) [inline readthedocs/templates/account/email.html]: nit: I think we have been using `"` for these blocks.

File: readthedocs/templates/account/email.html [hunk 3]
```diff
@@ -9,68 +9,104 @@
...
+        }
+      });
+    }
+  })();
+
+  (function() {
+    var primaryButton = document.querySelector("button[name=action_primary]");
+    var sendVerificationButton = document.querySelector("button[name=action_send");
+
+    {% if user.emailaddress_set.all %}
```
Comment (humitos) [inline readthedocs/templates/account/email.html]: nit: use `.exists` here instead of `.all`.

**Comment by humitos** [comment]
> Currently, I have implemented this on client side, this should also be from server side. I was able to send 3-4 email verification email for a verified email.

We could, but I think this is not really important in this case.

File: readthedocs/templates/account/email.html (hunk 1 above)
Comment (dojutsu-user) [reply readthedocs/templates/account/email.html]: Thank you.
I have pushed the changes.

File: readthedocs/templates/account/email.html (hunk 3 above)
Comment (dojutsu-user) [reply readthedocs/templates/account/email.html]: Thank you.
I have pushed the changes.

**Comment by dojutsu-user** [comment]
@davidfischer 
> I think things like this get a lot easier once we have a better UI framework in place.

Are there any plans to work on the UI part of RTD?

**Comment by davidfischer** [comment]
> Are there any plans to work on the UI part of RTD?

Yes there are. This has been something I've been pushing on for a while but it's actually a pretty huge task.

**Comment by dojutsu-user** [comment]
@davidfischer 
I also want the UI to improve and would like to collaborate on that whenever the team decides to work on the UI part.

**Comment by humitos** [comment]
I'm merging the PR now since it's tested and we agree on it.

**Pull request merged**